File: tiger_app/tiger_app/report/chutney_report/chutney_report.js
// Copyright (c) 2016, amran and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Chutney Report"] = {
	"filters": [
		{
            "fieldname":"fruit_name",
            "label": __("Fruit Name"),
            "fieldtype": "Data",
			"default": "",
			"reqd": 1
        }
	]
};

File: tiger_app/tiger_app/report/chutney_report/chutney_report.py
# Copyright (c) 2013, amran and contributors
# For license information, please see license.txt

# import frappe
from __future__ import unicode_literals
import frappe
from frappe import _


def execute(filters=None):
	columns, data = ["Fruit Name","Point"], []
	data = get_all()
	return columns, data

def get_all():
	return frappe.db.sql(""" 
		SELECT
		 fruit_name, point
		FROM
		`tabChutney`
		""")

